### ISSUE TITLE
java-openliberty: Slim down server feature definitions

### ISSUE THREAD

**Issue by awisniew90** [appsody/stacks#592]
Currently, the server configuration for the default template includes the full microProfile-3.2 feature. This can be slimmed down to improve startup time (i.e. mpHealth-2.0, mpMetrics-2.0, jaxrs).  In addition we should add an IT to test the application endpoint and make sure jaxrs is functioning and available.

**Comment by yeekangc** [comment]
Fwiw, I am on the fence on this one.  This should improve startup time.  Then, the template will be less usable for developers since more likely than not, they will need to add more features.  It's also a deviation from our ideal/opinionated template too.

**Comment by scottkurz** [comment]
Maybe we should approach this more broadly as "improve 'run' startup time".    We could explore SCC.   Maybe there's also some of the LTPA/SSL config we could cache?    My startup looks like:

```
[Container] [INFO] [AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
[Container] [INFO] [AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
[Container] [INFO] [AUDIT   ] CWWKS4104A: LTPA keys created in 4.579 seconds. LTPA key file: /opt/ol/wlp/usr/servers/defaultServer/resources/security/ltpa.keys
[Container] [INFO] [AUDIT   ] CWPKI0803A: SSL certificate created in 4.815 seconds. SSL key file: /opt/ol/wlp/usr/servers/defaultServer/resources/security/key.jks
[Container] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://581c32c67286:9080/openapi/ui/
[Container] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://581c32c67286:9080/ibm/api/
[Container] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://581c32c67286:9080/metrics/
[Container] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://581c32c67286:9080/jwt/
[Container] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://581c32c67286:9080/health/
[Container] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://581c32c67286:9080/openapi/
[Container] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://581c32c67286:9080/
[Container] [INFO] [AUDIT   ] CWWKZ0001I: Application starter-app started in 5.100 seconds.
```

Besides the feature set, (which I'm not saying we've ruled out thinning down), we could look at:

1. turning off app monitor of dropins?   (Could that save us any time).
2. key.jks generation
3. ltpa.keys generation
4. SCC

**Comment by scottkurz** [comment]
Some interesting ideas, but on balance not helpful to have such an open-ended issue open, so closing for now.
